pcdsdevices/fms.py

```diff
@@ -0,0 +1,259 @@
+CXI:SETRA:01:SAMPLE_TIME_RBV 60 monitor
+CXI:SETRA:01:HOLD_TIME_RBV 60 monitor
+CXI:SETRA:01:SAMPLE_MODE_RBV 60 monitor
+CXI:SETRA:01:MASS_MODE_RBV 60 monitor
+CXI:SETRA:01:RECIPE_MODE_RBV 60 monitor
+CXI:SETRA:01:RECORD_REQ 60 monitor
+CXI:SETRA:01:RH 60 monitor
+CXI:SETRA:01:BP 60 monitor
+CXI:SETRA:01:TEMP 60 monitor
+CXI:SETRA:01:DATE 60 monitor
+CXI:SETRA:01:TIME 60 monitor
+CXI:SETRA:01:TVOC 60 monitor
+CXI:SETRA:01:CO2 60 monitor
+CXI:SETRA:01:FLOW_RATE 60 monitor
+CXI:SETRA:01:SAMPLE_DURATION 60 monitor
+CXI:SETRA:01:CH0_SIZE 60 monitor
+CXI:SETRA:01:CH0_MASS 60 monitor
+CXI:SETRA:01:CH1_SIZE 60 monitor
+CXI:SETRA:01:CH1_MASS 60 monitor
+CXI:SETRA:01:CH2_SIZE 60 monitor
+CXI:SETRA:01:CH2_MASS 60 monitor
+CXI:SETRA:01:CH3_SIZE 60 monitor
+CXI:SETRA:01:CH3_MASS 60 monitor
+CXI:SETRA:01:CH4_SIZE 60 monitor
+CXI:SETRA:01:CH4_MASS 60 monitor
+CXI:SETRA:01:CH5_SIZE 60 monitor
+CXI:SETRA:01:CH5_MASS 60 monitor
+"""
+
+
+from ophyd import Component as Cpt, Device, EpicsSignalRO
+from ophyd import FormattedComponent as FCpt, EpicsSignalRO, Device
+
+class Setra5000(Device):
+    Timestamp = Cpt(EpicsSignalRO, ':TIME', kind='hinted')
+    Date = Cpt(EpicsSignalRO, ':DATE', kind='hinted')
+
+    SampleState = Cpt(EpicsSignalRO, ':SAMP_STATE', kind='hinted')
+    SampleDuration = Cpt(EpicsSignalRO, ':SAMPLE_TIME_RBV', kind='hinted')
+    SampleMode = Cpt(EpicsSignalRO, ':SAMPLE_MODE_RBV', kind='hinted')
+    MassMode = Cpt(EpicsSignalRO, ':MASS_MODE_RBV', kind='hinted')
+    
+    Temperature = Cpt(EpicsSignalRO, ':TEMP', kind='hinted')
+    RH = Cpt(EpicsSignalRO, ':RH', kind='hinted')
+    BarometricPressure = Cpt(EpicsSignalRO, ':BP', kind='hinted')
+    FlowRate = Cpt(EpicsSignalRO, ':FLOW_RATE', kind='hinted')
+    CO2 = Cpt(EpicsSignalRO, ':CO2', kind='hinted')
+    TVOC = Cpt(EpicsSignalRO, ':TVOC', kind='hinted')    
+
+    '''
+    ch0_size = Cpt(EpicsSignalRO, ':CH0_SIZE', kind='normal')
+    ch0_mass = Cpt(EpicsSignalRO, ':CH0_MASS', kind='normal')
+    ch1_size = Cpt(EpicsSignalRO, ':CH1_SIZE', kind='normal')
+    ch1_mass = Cpt(EpicsSignalRO, ':CH1_MASS', kind='normal')
+    ch2_size = Cpt(EpicsSignalRO, ':CH2_SIZE', kind='normal')
+    ch2_mass = Cpt(EpicsSignalRO, ':CH2_MASS', kind='normal')
+    ch3_size = Cpt(EpicsSignalRO, ':CH3_SIZE', kind='normal')
+    ch3_mass = Cpt(EpicsSignalRO, ':CH3_MASS', kind='normal')
+    ch4_size = Cpt(EpicsSignalRO, ':CH4_SIZE', kind='normal')
+    ch4_mass = Cpt(EpicsSignalRO, ':CH4_MASS', kind='normal')
+    ch5_size = Cpt(EpicsSignalRO, ':CH5_SIZE', kind='normal')
+    ch5_mass = Cpt(EpicsSignalRO, ':CH5_MASS', kind='normal')
+    Total_Mass = Cpt(EpicsSignalRO, ':TOT_MASS', kind='normal')
+    '''
+
+    # kind, doc are important keywords for later
+    # kind=hinted, normal, config, omitted
+
+
+
+ #Power Distribution Units: Temperature, Humidity, and Load Feed
+
+#PDU_Temp1: Will be for racks with 2 PDUs, with total of 3 sensors for monitoring purposes
+class PDU_Temp1(Device):
```

Comment:
```suggestion
class PDU_Temp1(Device):
    """ A rack with 2 PDUs, with total of 3 sensors for monitoring purposes """
```

Moving these comments into the class as a docstring will let us connect this information to the device when it's created later on.  This should be done throughout the file.